guidance/library/_json.py

```diff
@@ -285,7 +287,7 @@ def _process_anyOf(
 
 @guidance(stateless=True)
 def _process_enum(lm, *, options: Sequence[Mapping[str, Any]]):
-    # options will come in as python objects, so we need to convert to (compact) JSON
+    # TODO: can we support a whitespace-flexible version of this?
```

Comment:
Would be nice. If you saw my 'perplexity' notebook in the Cookbook, just reformatting the produced JSON massively reduced the model perplexity. While high perplexity is not directly indicative of a poor generation, it is a hint.